questions-js1/6.md

```diff
@@ -23,4 +23,5 @@ b) What will `result2` evaluate to? Explain your answer
 c) Try to summarise the main difference between `logSum` and `calculateSum`
 
 
-{YOUR ANSWERS HERE}
\ No newline at end of file
+{result 1 will valuate as the result of a + b wich is 10 + 32 =42.THe result will be 42
+result 2 will be: "10 + 32" because in this case is not asking to calculate but to input the values}
```

Comment:
you should try them both by running them in `node`, you may be surprised at the result!